src/app/examples/nativescript-only/list-header-and-footer/list-header-and-footer-example.component.tns.html

```diff
@@ -0,0 +1,18 @@
+<kirby-list [items]="items"
+        shape="rounded">
+        <StackLayout *kirbyListHeader>
+            <Label text="This is the header" textWrap="true" class="header"></Label>
+        </StackLayout>
+
+  <kirby-list-flex-item *kirbyListFlexItem="let item">
```

Comment:
Simplify the example by removing `<kirby-list-flex-item>` and `<kirby-list-cell>` wrappers and just applying `*kirbyListItem` directive to a `StackLayout` itself (as you already do for header/footer):
```suggestion
  <StackLayout *kirbyListItem>
          <Label [text]="item.title"></label>
        </StackLayout>
```

Reply:
Isn't `*kirbyListItem` going to be deprecated? Or are we going to differentiate on `*kirbyListItem` and `*kirbyListFlexItem` as such?

Reply:
It's a left over from the performance issue just before the release - but no, `*kirbyListItem` is not going to be deprecated :)